src/api.rs

```diff
@@ -57,6 +57,13 @@ fn populate_span_with_admission_request_data(adm_req: &AdmissionRequest) {
 fn populate_span_with_policy_evaluation_results(validation: &ValidationResponse) {
     Span::current().record("allowed", &validation.allowed);
     Span::current().record("mutated", &validation.patch.is_some());
+    if let Some(status) = &validation.status {
+        Span::current().record("response_code", &status.code.unwrap_or_default());
+        Span::current().record(
+            "response_message",
+            &status.message.clone().unwrap_or_default().as_str(),
+        );
```

Comment:
I think is better to `record` only if they have a `Some`, something like (pseudorust):

```rust
if let Some(response_code) = status.code {
  Span::current().record("response_code", response_code);
}
if let Some(response_message) = status.message {
  Span::current().record("response_code", response_message);
}
```

This way, if they are not filled, they are kept as `tracing::field::Empty` as opposed to an empty string. This ensures that it's not recorded altogether, as opposed to being recorded as an empty string.

Reply:
good point! done in https://github.com/kubewarden/policy-server/commit/04ace1da7eca104f51d0e81634fa8b2a734bf1d1